testssl.sh

```diff
@@ -5042,6 +5167,7 @@ run_protocols() {
                     latest_supported_string="SSLv3"
                fi
                add_tls_offered ssl3 yes
+               set_grade_cap "B" "SSLv3 is offered"
```

Comment:
Just a comment: I thought this would be an error. But it isn't (https://github.com/ssllabs/research/wiki/SSL-Server-Rating-Guide#protocol-support)

Reply:
Likely I'll adjust that

Reply:
It's an old rule yes: https://github.com/ssllabs/research/wiki/SSL-Server-Rating-Guide#changes-in-2009h-30-october-2014

I also think we could adjust it. B is way too high IMO

Reply:
The problem I see: if we touch the rating it's hard to finger point at SSLlabs